frontend/src/components/Avatar/Avatar.vue

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
+  name: 'Avatar',
+  props: {
+    imageUrl: {
+      type: String,
+      required: true
+    },
+    altText: {
+      type: String,
+      required: true
+    }
+  }
+})
+</script>
+
+<template>
+  <div class="flex justify-center items-center rounded-full overflow-hidden h-16 w-16 bg-gray-200">
```

Comment:
It would be great to add sizing presets, (small, medium, large). The current size can remain the default as medium.

Reply:
Added the presets in the Avatar.vue file and updated the Storybook component to reflect these changes!